pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,275 @@
+    - **Key Pair** - The SSH Key Pair used to access your nodes. Please refer to this guide on [how to create a SSH KeyPair and add it to your Public Cloud project](https://help.ovhcloud.com/csm/en-public-cloud-compute-getting-started?id=kb_article_view&sysparm_article=KB0051009).
+    - **Security Group** - Existing security groups from our Public Cloud project.
+    - **Availability Zone** - Only `nova` is supported at the moment.
+    - **Floating IP Pools** - Only `Ext-Net` is supported at the moment.
+    - **Networks** - Public Cloud Network to attach your nodes
+    - **SSH user ID** - Username that will be used to access your nodes
+
+
+8. Use the **Cluster Configuration** to choose the version of Kubernetes that will be installed, what network provider will be used and if you want to enable project network isolation. For help configuring the cluster, refer to the [RKE2 cluster configuration reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration).
+Use Member Roles to configure user authorization for the cluster. Click Add Member to add users that can access the cluster. Use the Role drop-down to set permissions for each user.
+10. Click **Create**.
+
+
+### Import an existing Kubernetes cluster
+
+For organizations with pre-existing Kubernetes clusters, Rancher simplifies integration. Import your clusters seamlessly, wherever they are deployed, allowing Rancher to take over the management responsibilities. This process facilitates the transition to Rancher without disrupting your existing infrastructure.
+
+You can refer to the official Rancher documentation on how to [Register Existing Cluster](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters).
+
+#### Import an existing OVHcloud [Managed Kubernetes Service](https://www.ovhcloud.com/en-gb/public-cloud/kubernetes/) cluster
+
+If you already use our [OVHcloud Managed Kubernetes Service](https://www.ovhcloud.com/en-gb/public-cloud/kubernetes/) , you can easily import existing cluster.
+The workflow is similar to the one described on the official Rancher documentation on [how to register a cluster](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters#registering-a-cluster) and it takes only few minutes.
+
+1. From the Rancher Homepage, click on **Import Existing**
+![Rancher Homepage](images/rancher-homepage.png)
+2. Select **Generic**
+![Import Cluster](images/import-cluster.png)
+3. Set the Cluster Name (it is not mandatory to match the name of your existing MKS cluster) then click on **Create**
```

Comment:
About the cluster name, 
![image](https://github.com/ovh/docs/assets/2543381/51adb980-c8a1-4358-9c28-06a4de96d3a7)
it appears that we can't name the cluster like on OVHcloud.

Please add the restrictions about the naming.